src/qutip_cupy/__init__.py

```diff
@@ -1 +1,21 @@
 from .version import version as __version__
+
+from qutip.core import data
+from .cupy_dense import CuPyDense
+
+def dense_from_new(newarray):
+    
+    dense_np = data.Dense(newarray._cp.tolist())
+    return dense_np
```

Comment:
This is quite inefficient - you probably want something like `data.Dense(cupy.asnumpy(array), copy=False)`.  Also, it would be better if these functions have descriptive names - what does `new` refer to?

Reply:
Yeahp I changed the naming of the variables everywhere to make them more descriptive. And changed the casting method to `cp.asnumpy` iit also powert the `toarray`method which was undefined till that point.